install.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+#/home/shiv1/pypy/pypy3.8-v7.3.7-linux64/bin/python3.8
```

Comment:
Todo(shiv): Remove 
(I tried running the legate core with pypy, this didn't end up working)